### PR TITLE
script run for traffic gen

### PR DESCRIPTION
**Pull request by murpheux**

### CODE REVIEW

File: run-traffic.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+BASE_URL='https://eagle-test.pathfinder.gov.bc.ca'
```
Comment (ActionAnalytics) [inline run-traffic.sh:3]: keep this

File: run-traffic.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+PROJECT_PREFIX='p/58851197aaecd9001b8227cc/project-details;currentPage=1;pageSize=10;sortBy=-dateAdded;ms='
```
Comment (ActionAnalytics) [inline run-traffic.sh:4]: get rid of this

File: run-traffic.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+PROJECTS=(1580750409346 1580750409346 1580750585604 1580750713545)
```
Comment (ActionAnalytics) [inline run-traffic.sh:5]: convert this to a PATH variable per the ticket

File: run-traffic.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+DOCUMENT_PREFIX='api/document'
```
Comment (ActionAnalytics) [inline run-traffic.sh:7]: get rid of documents entirely

File: run-traffic.sh (hunk 1 above)
Comment (murpheux) [reply run-traffic.sh:3]: Done

File: run-traffic.sh (hunk 2 above)
Comment (murpheux) [reply run-traffic.sh:4]: Done

File: run-traffic.sh (hunk 3 above)
Comment (murpheux) [reply run-traffic.sh:5]: Done

File: run-traffic.sh (hunk 4 above)
Comment (murpheux) [reply run-traffic.sh:7]: Done

**Pull request merged**